packages/discord.js/typings/index.d.ts

```diff
@@ -3379,13 +3377,19 @@ export class ThreadMemberManager extends CachedManager<Snowflake, ThreadMember,
   public remove(id: Snowflake | '@me', reason?: string): Promise<Snowflake>;
 }
 
-export class UserManager extends CachedManager<Snowflake, User, UserResolvable> {
-  private constructor(client: Client, iterable?: Iterable<RawUserData>);
+export class UserManager<Ready extends boolean> extends BaseManager {
```

Comment:
So we can use it without having to specify:
```suggestion
export class UserManager<Ready extends boolean = boolean> extends BaseManager {
```